modules/apps/object/object-api/bnd.bnd

```diff
@@ -10,6 +10,7 @@ Export-Package:\
 	com.liferay.object.definition.notification.term.util,\
 	com.liferay.object.deployer,\
 	com.liferay.object.display.context,\
+	com.liferay.object.dsl,\
```

Comment:
@joseabelenda 

Does use `com.liferay.object.petra.sql.dsl,\` to keep the same path used in **object-service** make sense for you?

I mean, is there a reason why you opted to define as `com.liferay.object.dsl`

Thank you!

Reply:
Ok. It makes sense.
Thank you.